igvc_navigation/src/mapper/line_layer.cpp

```diff
@@ -445,17 +448,24 @@ void LineLayer::debugPublishPC(ros::Publisher &pub, const cv::Mat &mat, geometry
 
 void LineLayer::initCostTranslationTable()
 {
-  cost_translation_table_.resize(256);
-  cost_translation_table_[0] = 0;      // NO obstacle
-  cost_translation_table_[253] = 99;   // INSCRIBED obstacle
-  cost_translation_table_[254] = 100;  // LETHAL obstacle
-  cost_translation_table_[255] = -1;   // UNKNOWN
+  constexpr int8_t free_space_msg_cost = 0;
+  constexpr int8_t inflated_msg_cost = 99;
+  constexpr int8_t lethal_msg_cost = 100;
+  constexpr int8_t unknown_msg_cost = -1;
+
+  cost_translation_table_.resize(std::numeric_limits<uchar>::max() + 1);
+
+  cost_translation_table_[costmap_2d::FREE_SPACE] = free_space_msg_cost;                 // NO obstacle
+  cost_translation_table_[costmap_2d::INSCRIBED_INFLATED_OBSTACLE] = inflated_msg_cost;  // INSCRIBED obstacle
+  cost_translation_table_[costmap_2d::LETHAL_OBSTACLE] = lethal_msg_cost;                // LETHAL obstacle
+  cost_translation_table_[costmap_2d::NO_INFORMATION] = unknown_msg_cost;                // UNKNOWN
 
   // regular cost values scale the range 1 to 252 (inclusive) to fit
   // into 1 to 98 (inclusive).
-  for (int i = 1; i < 253; i++)
+  for (int i = 1; i <= costmap_2d::INSCRIBED_INFLATED_OBSTACLE - 1; i++)
   {
-    cost_translation_table_[i] = char(1 + (97 * (i - 1)) / 251);
+    // NOLINTNEXTLINE
```

Comment:
What were you considering when adding the NOLINTs? I don't quite get why these were necessary when they could be easily solved with a couple variables.

Reply:
I was thinking that it would be very annoying to make all these numbers constants and like wouldn't really help much with readability, and it'd be like changing code for the sake of passing the linter.